packages/components/date-picker/package.json

```diff
@@ -40,22 +40,22 @@
     "react-dom": ">=18"
   },
   "dependencies": {
-    "@internationalized/date": "^3.5.4",
+    "@internationalized/date": "3.5.5",
```

Comment:
The type of `DateValue` is different between 3.5.4 and 3.5.5, so I have fixed it to 3.5.5.
https://github.com/adobe/react-spectrum/pull/6076